src/ga4gh/vrs/__init__.py

```diff
@@ -23,4 +23,4 @@
 
 logging.basicConfig(
```

Comment:
Libraries should not initialize logging. This should be left to applications. Unexpected or inconsistent behavior is likely with this setup. If I left this here, it must've been during debugging. Up to you whether you want to address it for this PR since it's out of scope.

Reply:
I'll create a new issue for this